user-guide/Standard_Apps/Monitoring_Solutions/DITT/Open_Putty_with_DITT.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: Open_Putty_with_DITT
+---
+
+# Opening PuTTY using DITT
+
+To add a button to open PuTTY on the DITT visual overview:
```

Comment:
Should PuTTY be installed before you add the button? If yes, where can users find the installation package for this?

Reply:
Hello @MariekeGO 
I put the PuTTy download link